woocommerce/woocommerce-ios
# [HACK Week]Add "Fakes.framework" documentation

closes #3096

# Why

Now that the `Fakes.framework` is [integrated](#3813) in the project, is more than fair to add propper documentation to our repository so old and newcomers can understand better how we work.

# Testing Steps

See [live view](https://github.com/woocommerce/woocommerce-ios/blob/3d303fd8c143c33977a580665ce982bed7d5a5da/docs/README.md) and tap on the fakeable links.

Update release notes:
